wqflask/tests/wqflask/show_trait/test_export_trait_data.py

```diff
@@ -3,10 +3,72 @@
 from wqflask.show_trait.export_trait_data import dict_to_sorted_list
 from wqflask.show_trait.export_trait_data import cmp_samples
 from wqflask.show_trait.export_trait_data import export_sample_table
+from wqflask.show_trait.export_trait_data import get_export_metadata
+
+
+class MockGeneral(object):
+    def __init__(self, obj):
+        for key, value in obj.items():
+            setattr(self, key, value)
```

Comment:
**praise:** This is really cool

I should re-use this snip else where when mocking things!